packages/discordbot/package.json

```
@@ -0,0 +1,27 @@
+{
+    "name": "discordbot",
+    "version": "1.0.0",
+    "description": "",
+    "main": "dist/index.js",
+    "scripts": {
+        "build": "tsc -w",
```

build script runs in watch mode
```suggestion
        "build": "tsc",
```